totalopenstation/output/__init__.py

```diff
@@ -2,6 +2,35 @@
 
 __all__ = ["tops_csv", "tops_dxf", "tops_dat", "tops_sql", "tops_txt", "tops_geojson"]
 
+class Builder:
+
+    def __init__(self, data):
+        """Init method which **should** be overridden in the child class
```

Comment:
The methods do nothing at all so both **must** be overridden.

Reply:
Modified in PR #113